src/modules/dashboard/components/confirmation-dialog.vue

```diff
@@ -33,12 +38,16 @@ const props = defineProps({
   isDialogVisible: { type: Boolean, default: false },
 });
 
+const { title, message, dialogType } = useDialog();
+
+const buttonRole = computed(() =>
+  dialogType.value === DialogTypeEnum.DELETE ? 'delete' : 'default',
```

Comment:
Jeżeli dialogTypeEnum posiada delete oraz default (wartosci) to rownie dobrze mozna je tutaj wstawic zamiast stringów pisanych z palca